msg/CMakeLists.txt

```diff
@@ -93,6 +93,8 @@ set(msg_files
 	onboard_computer_status.msg
 	optical_flow.msg
 	orbit_status.msg
+	output_control.msg
+	output_feedback.msg
```

Comment:
This topic can be removed for the initial PR - but keep the idea of feedback in the back of your collective minds for future work.